dotnet-desktop-guide/net/winforms/whats-new/index.md

```diff
@@ -14,8 +14,7 @@ There are a few breaking changes you should be aware of when migrating from .NET
 ## Enhanced features
 
 - Microsoft UI Automation patterns work better with accessibility tools like Narrator and Jaws.
-- Improved performance and reliability.
```

Comment:
We did improve performance, so I think it is worth keeping. Just not "reliability" bit :)